GVRf/Framework/framework/src/main/jni/objects/textures/render_texture.cpp

```diff
@@ -23,215 +23,438 @@
 #include "eglextension/msaa/msaa.h"
 
 namespace gvr {
-RenderTexture::RenderTexture(int width, int height, GLTexture* tex) :
+GLFrameBuffer *RenderTexture::renderTexture_gl_read_buffer = nullptr;
```

Comment:
we need to store readFBO somewhere, we don't have render-texture object in "readRenderResultNative()" as textures are created by oculus and it doesn't make sense to create our own render-texture as we just need texture.